common.pri

```diff
@@ -44,5 +44,28 @@ CONFIG += warn_on
 QMAKE_CXXFLAGS += -Wextra
 QMAKE_CXXFLAGS_DEBUG += -Wextra
 
-# QuaZIP: use as static library
-DEFINES += QUAZIP_STATIC
+# enable pkgconfig on linux when unbundling
+!isEmpty(UNBUNDLE) {
+    unix:CONFIG += link_pkgconfig
+}
+
+# If UNBUNDLE contains "all", unbundle all dependencies
+contains(UNBUNDLE, all) {
+    UNBUNDLE = (all) quazip fontobene-qt5
+}
+
+# QuaZIP configuration
+contains(UNBUNDLE, quazip) {
+    DEFINES += SYSTEM_QUAZIP
+} else {
+    DEFINES += QUAZIP_STATIC
+}
+
+# add linking information to defines
+isEmpty(UNBUNDLE) {
+    DEFINES += "LINKING_TYPE=\"\\\"static\\\"\""
+    DEFINES += "UNBUNDLE=\"\\\"-\\\"\""
+} else {
+    DEFINES += "LINKING_TYPE=\"\\\"dynamic\\\"\""
+    DEFINES += "UNBUNDLE=\"\\\"$${UNBUNDLE}\\\"\""
+}
```

Comment:
These defines are only needed for the `Application` class, right? Then I'd prefer to move them to `common.pro` instead to keep their scope as small as possible.

Reply:
Yep, right now that's the only place where these defines are used. I'll move them.

Reply:
Now these defines are in both files, `common.pri` and `common.pro` :wink:

Reply:
Gahhh, rebasing was a bit messy due to conflicts. I resolved one of them, but it seems that git auto-resolved the other conflict the wrong way.

Reply:
So could you fix it now? :wink:

Reply:
I thought I did, but edited the wrong location again... It doesn't help that both files are called `common` :see_no_evil: 

Should be fixed now.